javascript/src/test/pages/Courses/CourseShow.test.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { waitFor, render } from "@testing-library/react";
+import CourseShow from "main/pages/Courses/CourseShow";
+import { useParams} from "react-router-dom";
+
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"), // use real functions 
+  useParams: jest.fn(), // except for just this one
+}));
+
+import CourseDetail from "main/components/Courses/CourseDetail";

```

Comment:
Can we keep all the imports together by moving this import to be somewhere in lines 1-5?

Reply:
THis will be solved in the next commit